decidim-elections/app/assets/config/decidim_votings_polling_officer_zone_manifest.js

```diff
@@ -0,0 +1,2 @@
+ //= link decidim/votings/polling_officer_zone/new-closure.js
+ //= link decidim/votings/polling_officer_zone/edit-closure.js
```

Comment:
Not an expert, but I think this is not needed anymore since the `webpack` migration?